be/src/connector/lake_connector.cpp

```diff
@@ -282,7 +282,7 @@ Status LakeDataSource::get_next(RuntimeState* state, ChunkPtr* chunk) {
             SCOPED_TIMER(_expr_filter_timer);
             size_t nrows = chunk_ptr->num_rows();
             _selection.resize(nrows);
-            _not_push_down_predicates.evaluate(chunk_ptr, _selection.data(), 0, nrows);
+            RETURN_IF_ERROR(_not_push_down_predicates.evaluate(chunk_ptr, _selection.data(), 0, nrows));
```

Comment:
It's a bug if here error ocurrs, so FE should avoid illegal plan from being delivered to BE